source/_docs/z-wave/controllers.markdown

```diff
@@ -63,6 +63,19 @@ The alternative to a stick is a hub that supports Z-Wave. Home Assistant support
 
 ## Controller Notes
 
+### 800 Series Controllers
+
+Some features are not supported with 800 series controllers yet.
+Unsupported
+ - Long Range
+ - NVM Backup/Restore 
+Supported
+ - Device inclusion + exclusion (Basic and SmartStart)
+ - Setting Device Configuration Values
+ - Getting Device Configuration Values
+ - Command Classes: Multilevel Switch v2 and v4, Color Switch v1, and Config v1 and v2
+ - Device Heal
```

Comment:
This is not a controller-specific feature, also it was renamed to "rebuild routes".

Reply:
Renamed, not sure what you mean by controller specific feature. This PR adds documentation for what is supported by z-wave JS in regards to 800 series.

Reply:
Setting values, command classes, and (for the most part) rebuilding routes have nothing to do with the controller being 500, 700 or 800 series.